tailwind.config.ts

```diff
@@ -85,6 +85,10 @@ export default {
             transform: 'scale(1)',
           },
         },
+        pulse: {
```

Comment:
name it same like other hover animations and keyframes

Reply:
Done